src/status_im2/navigation/core.cljs

```diff
@@ -167,18 +167,11 @@
 
 (navigation/reg-button-pressed-listener
  (fn [id]
-   (cond
-     (= "dismiss-modal" id)
+   (if (= "dismiss-modal" id)
      (do
        (when-let [event (get-in views/screens [(last @state/modals) :on-dissmiss])]
          (re-frame/dispatch event))
        (dissmissModal))
-     (= "RNN.hardwareBackButton" id)
```

Comment:
do we not these for android devices back button?

Reply:
this was kinda a hack, there is a proper way to listen to navigate back button